lms/static/scripts/frontend_apps/components/dashboard/AssignmentActivity.tsx

```diff
@@ -44,6 +45,8 @@ export default function AssignmentActivity() {
     [students.data],
   );
 
+  useDocumentTitle(`${assignment.data?.title ?? 'Untitled assignment'}`);
```

Comment:
```suggestion
  useDocumentTitle(assignment.data?.title ?? 'Untitled assignment');
```

No need to use string interpolation here.